Resources/public/js/views/ez-notificationhubview.js

```diff
@@ -41,22 +41,11 @@ YUI.add('ez-notificationhubview', function (Y) {
          */
         _setListEventHandlers: function (list) {
             list.on('add', Y.bind(function (e) {
-                var oldHeight = this._getContainerHeight();
-
                 this._addNotificationView(e.model);
-                this._fireHeightChange(
-                    oldHeight, this._getContainerHeight()
-                );
             }, this));
 
             list.on('remove', Y.bind(function (e) {
-                var oldHeight = this._getContainerHeight(),
-                    newHeight = oldHeight;
-
-                newHeight -= this._removeNotificationView(e.model);
```

Comment:
since we don't need to do the math anymore, I guess you can remove the height computation from `_removeNotificationView` ?